Userland/DevTools/HackStudio/HackStudioWidget.h

```diff
@@ -81,6 +82,7 @@ class HackStudioWidget : public GUI::Widget {
 
     void update_current_editor_title();
     void update_window_title();
+    RefPtr<NewProjectDialog> m_new_project_dialog;
```

Comment:
this should be private

Reply:
Yes, with the above changes it can be. With my suboptimal solution this needed to be public.